src/gui/accountstate.cpp

```diff
@@ -94,6 +94,14 @@ void AccountState::setState(State state)
             _connectionStatus = ConnectionValidator::Undefined;
             _connectionErrors.clear();
         } else if (oldState == SignedOut && _state == Disconnected) {
+            // If we stop being voluntarily signed-out, try to connect and
```

Comment:
Nah, I just sneaked in this comment because I felt it was missing. It's technically unrelated.